### DISCUSSION TITLE
缠中说禅博客原文中的思考题

### DISCUSSION THREAD

**Discussion by zengbin93** [waditu/czsc#41] [Ideas]
* 日线图上的笔、线段，和一分钟图上的日中枢有什么关系？

**Comment by zengbin93** [comment]
## 日线图上的笔、线段，和一分钟图上的日中枢有什么关系？

**Comment by alxbj** [comment]
如果按递归级别，两者没关系...一分钟图上的“日”中枢，这个中枢也只是起了个“日”的名字而已...
如果以时间作为级别，那就是放大镜下的嵌套关系，不过这个关系对量化没有意义...

因此，想让他们的关系变得有意义，比如1分钟的onbar出现信号，用日级别的分析做过滤。就需要这两个级别判断独立且并行的在回调函数中调用....
